src/components/Modal/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import getModalStyles from '../../styles/getModalStyles';
+import CONST from '../../CONST';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import ModalSkeleton from './ModalSkeleton';
+
+const propTypes = {
```

Comment:
These propTypes can go into a new file and get imported in each of the `index.js` and `index.android.js` files